### DISCUSSION TITLE
I think it would be good if you could see ownership history of an NFT.

### DISCUSSION THREAD

**Discussion by jam2ja** [deso-protocol/dips#121] [Medium CIPs]
Yes. Agree, especially if friends are just buying NFTs from friends. Seems to be the case in many instances)

_Originally posted by @KrassensteinBitClout in https://github.com/bitclout/cips/issues/92#issuecomment-889420225_

**Comment by tijno** [comment]
@jam2ja can you edit the title so its short & to the point and move the rest to the comment.

**Comment by jam2ja** [comment]
Excuse that..I'm a very new user. To continue with this point. As a buyer, first thing i would be doing on an NFT social network when looking at an artist in consideration to buy, is to see if a person that bought art from that "artist" was friends of that person within the social network, for previous sales. It would be good for buyers to see this on an indicator. Supported by friends with purchase..tag. This might make fraud less prevalent on the site. If an artists value is created just out of friends buying/selling her/his art, this might be seen as whole sale fraud. If the blockchain would show the cycle? If unsuspecting buyer enters the market "club house room" and pays a large price, then looks back and sees that all the artists transactions prior to his were with friends on that site. I suspect that the US authorities would start to know about it, maybe they already do? I'd jump on this one quickly. Supported by friends with purchase..tag.  Ironically. I've heard bankers do this in the art world.. they buy art from unknown artists, they make them famous as a group, recycling the price in conjunction with the gallery/artist. The irony is/might be that when the artist dies, as the art is dependant on production, the art has little value. or maybe not? Food for thought..
